scripts/import/import_clinvar_xml

```diff
@@ -1458,21 +1509,40 @@ sub clean_old_data{
     $allele_syn_delete_sth->execute() || die "Could not delete entry id $allele_syn_id from allele_synonym\n";
   }
 
+  print "Deleting old variation feature, transcript variation and MTMP transcript variation\n";
+  # remove from variation_feature, transcript_variation and MTMP_transcript_variation
+  my $tv_del_sth = $dba->dbc()->prepare(qq[ select vf.variation_feature_id from variation_feature vf
+                                            left join source s on s.source_id = vf.source_id
+					    where s.name ='ClinVar';
+					    ]);
```

Comment:
If the left join is slow, we can instead fetch `source_id` with
```mysql
SELECT s.source_id FROM source s WHERE s.name = 'ClinVar'
```

and store it in a variable (e.g. `$source_id`) to provide it here:

```sql
SELECT vf.variation_feature_id FROM variation_feature vf
WHERE vf.source_id = $source_id
```